Fixing parameter desc + adding missing parameters (#5939)
DataDog/documentation: content/en/getting_started/integrations/prometheus.md

| `ssl_ca_cert`               | string                                  | optional  | none          | The path to the trusted CA used for generating custom certificates.                                                                                                                                                                                                  |
| `prometheus_timeout`        | integer                                 | optional  | 10            | Set a timeout in seconds for the Prometheus/OpenMetrics query.                                                                                                                                                                                                       |
| `max_returned_metrics`      | integer                                 | optional  | 2000          | The check limits itself to 2000 metrics by default. Increase this limit if needed.                                                                                                                                                                                   |

> | `bearer_token_auth`         | boolean                                 | optional  | false         | Set `bearer_token_auth` to `true` to add bearer token authentication header. Note: if `bearer_token_path` is not set, `/var/run/secrets/kubernetes.io/serviceaccount/token` is used as a default path                                                                |

Reviewer: ```suggestion
| `bearer_token_auth`         | boolean                                 | optional  | false         | Set `bearer_token_auth` to `true` to add a bearer token authentication header. **Note**: If `bearer_token_path` is not set, `/var/run/secrets/kubernetes.io/serviceaccount/token` is used as the default path.                                                                |
```

> | `bearer_token_path`         | string                                  | optional  | none          | The path to a kubernetes service account bearer token file, make sure the file exists and mounted correctly. Note: bearer_token_auth should be set to true to enable adding the token to HTTP headers for authentication.                                            |

Reviewer: ```suggestion
| `bearer_token_path`         | string                                  | optional  | none          | The path to a Kubernetes service account bearer token file (make sure the file exists and is mounted correctly). **Note**: Set `bearer_token_auth` to `true` to enable adding the token to HTTP headers for authentication.                                            |
```
